engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/FlinkStreamingProcessRegistrar.scala

```diff
@@ -70,18 +70,18 @@ class FlinkStreamingProcessRegistrar(compileProcess: (EspProcess, ProcessVersion
     usingRightClassloader(env) {
       register(env, compileProcess(process, processVersion), testRunId)
     }
-    initializeStateDescriptors(env)
+    //initializeStateDescriptors(env)
```

Comment:
why you commented it?

Reply:
I wonder if it's still valid... I added TODO for now...